draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -938,6 +1022,14 @@
       receiving a packet destined for the Service ID, the ingress router must
       refresh the Timer. The ingress router must send a BGP Withdraw UPDATE
       for the Service ID upon expiration of the Timer.</t>
+      <!-- XXX JMH - this text regarding RT-Constrain is very confusing.  RTC
+      operates on BGP routes having route target extended communities.  This
+      document isn't suggesting to use the existing route targets defined in
+      RFC 4360 and using that type/subtype is probably inappropriate.  It'd be
+      better to define a new extended community for this feature. 
+
+      Additionally it's not clear what these timers are and how they interact
+      with service ids. -->
```

Comment:
Will discuss with the co-authors on concrete text to add

Reply:
This was deleted in Linda's merge text in main.  Marking this resolved.